BME68X gas sensor driver using BSEC (#11103)
nrfconnect/sdk-nrf: applications/asset_tracker_v2/doc/sensor_module.rst

The sensor module supports integration with the BSEC signal processing library using the external sensors, internal convenience API.
If enabled, the BSEC library is used instead of the BME680 Zephyr driver to provide sensor readings from the BME680 for temperature, humidity, and atmospheric pressure.
In addition, the BSEC driver provides an additional sensor reading, indoor air quality (IAQ), which is a metric given in between 0-500 range, which estimates the air quality of the environment.

> In the beginning, the IAQ will show as 50 (good air), but is automatically calibrated over time.

Reviewer: ```suggestion
In the beginning, the IAQ shows 50 (good air), but it is automatically calibrated over time.
```
